src/pages/meta-search.vue

```diff
@@ -9,9 +9,9 @@
 
       <i18n path="meta-search-page.intro" tag="p" class="mb-4">
         <template #link>
-          <a aria-label="sources" href="/sources">{{
+          <NuxtLink aria-label="sources" :to="localePath('/sources')">{{
```

Comment:
If we're already here, can we go ahead and translate the `sources` string in the aria-label if you don't mind?